src/matlab/runRRTIKServer.m

```diff
@@ -0,0 +1,38 @@
+s = IKServer();
```

Comment:
Is this file, runRRTIKServer.m called from someplace else?  I don't see any references to it in this pull request.